lms/static/js/spec/edxnotes/helpers.js

```diff
@@ -169,6 +249,13 @@ define(['underscore'], function(_) {
         getChapter: getChapter,
         getSection: getSection,
         getUnit: getUnit,
-        getDefaultNotes: getDefaultNotes
+        getDefaultNotes: getDefaultNotes,
+        stringStartsWith: stringStartsWith,
```

Comment:
I think we are not using this method anywhere else. If that's the case then we can remove this from here.